unyt/tests/test_unyt_array.py

```diff
@@ -1891,10 +1888,10 @@ def test_electromagnetic():
     P_cgs = I_cgs * I_cgs * R_cgs
     assert_equal(P_mks.units.dimensions, dimensions.power)
     assert_equal(P_cgs.units.dimensions, dimensions.power)
-    with pytest.raises(UnitsNotReducible):
-        P_cgs.in_cgs()
-    with pytest.raises(UnitsNotReducible):
-        P_mks.in_cgs()
+    assert_almost_equal(P_cgs.in_cgs(), P_cgs)
+    assert_almost_equal(P_mks.in_cgs(), P_cgs)
+    assert_almost_equal(P_cgs.in_mks(), P_mks)
```

Comment:
It looks like this same test is repeated twice?

Reply:
thanks!